changelog/Microsoft.AAD.Reporting.json

```
@@ -1,6 +1,104 @@
 {
   "changelog": [
-      {
+    {
+      "ChangeList": [
+          {
+              "Id": "244631fe-cf8c-11eb-995f-5903f88189f5",
```

All changelog objects in a single ChangeList should have the same Id value.
```suggestion
              "Id": "2443cc12-cf8c-11eb-995f-5903f88189f5",
```